app/pb_kits/playbook/pb_contact/docs/_contact_default.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= pb_rails("contact", props: {
+  contact_type: "cell",
+  value: "349-185-9988",
```

Comment:
Not saying I'd prefer this, but how does `contact_value` sound instead of `value`?  `value` is pretty generic.

Reply:
I guess the kit is `"contact"` though. Hmmm. Should `contact_type` just be `type` then?

Reply:
They should be consistent. I'll update one of them. I think I like contact_value...but let me look at the react kit as well